api-tests/specs/practitioner-service/createperson.js

```diff
@@ -2,15 +2,16 @@ const requests = require('../../data/practitioner-service/createperson')
 const conf = require('../../config/conf')
 
 beforeEach(function () {
-    baseRequest = request(conf.BASE_URL)
+    console.log(conf.baseUrl)
+    baseRequest = request(conf.baseUrl)
 })
 
 describe('As a user with Create Person permission, I want to have my create person request validated by the system, So that I cannot create an invalid person', function () {
 
     it('When I submit an API request to create a Person with a value for all mandatory fields and no fields exceed their specified maximum length, Then a new Person record is persisted in the system with a unique identifier And I receive a success acknowledgement', async () => {
         const response = await baseRequest.post('/practitioner').send(requests.validPerson);
         expect(response.text).to.contain("id")
-        expect(response.status).to.equal(HttpStatus.CREATED)
+        expect(response.status).to.equal(201)
```

Comment:
better to leave it as HttpStatus.CREATED ?

Reply:
Definitely. Updated